security/old/models.py

```diff
@@ -0,0 +1,421 @@
+import json
```

Comment:
You should import this file in security/models.py. Because django must somehow find models related to migrations. 

qa check had to find missing migrations